lib/linkerd2-trace-context/src/layer.rs

```diff
@@ -180,3 +185,23 @@ where
         Ok(Async::Ready(inner))
     }
 }
+
+fn request_labels<Body>(labels: &mut HashMap<String, String>, req: &http::Request<Body>) {
```

Comment:
Can these maps be keyed on `&'static str` to avoid all of the needless `to_strings`?

Reply:
they can be but we have to eventually convert the keys to String to give it to prost.